java/java.lsp.server/vscode/README.md

```diff
@@ -21,80 +21,39 @@
 
 -->
 
+This is a technology preview of [Apache NetBeans](http://netbeans.org)
+based extension for VSCode. Use it to get all the _goodies of NetBeans_
+via the VSCode user interface! Run on __JDK8__[1], __JDK11__, __JDK15__, etc. 
```

Comment:
Better to use "__JDK8__[*]" instead of "__JDK8__[1]" - when rendered, the original text looks like "JDK81"

Reply:
Here is your wish: 9036a26